moira.py

```diff
@@ -54,13 +65,13 @@ def _load_help(handle):
     and return values into and out of dictionaries and into and out of
     tuples.
     """
-    help_string = ', '.join(query('_help', handle)[0]).strip()
+    help_string = ", ".join(query("_help", handle)[0]).strip()
```

Comment:
Actually, throughout, why the change to double quotes?

Reply:
Formatter